lua/neogit/lib/git/rebase.lua

```diff
@@ -58,6 +59,56 @@ function M.onto(start, newbase, args)
   end
 end
 
+function M.reword(commit, message, args)
```

Comment:
For new functions, I always add type annotations. A lot of the codebase predates their use, but every little bit helps :)

Reply:
added